db/database_sql/Makefile

```
@@ -0,0 +1,28 @@
+DBNAME:=golang_study
+DATASOURCE:=root:password@tcp(localhost:3306)/$(DBNAME)
```

githubのgooseだと必要でした。
```suggestion
DATASOURCE:=root:password@tcp(localhost:3306)/$(DBNAME)?parseTime=true
```